include/mipi_syst/platform.h

```diff
@@ -13,18 +13,38 @@
 extern "C" {
```

Comment:
was this file added by you? if yes, it should not be here, put it under zephyr to distinguish it from the upstream code.

Reply:
@nashif the file(library/include/mipi_syst.h.in) in syst library will include the platform.h in this way `#include "mipi_syst/platform.h", IMHO it's not public API for logging sub system, so it's not reasonable put it under include/logging; also there is no include folder under subsys/logging, and I'm not sure if adding include folder under subsys/logging is acceptable or not. And putting it here, just don't want to break anything for the logging system.
But anyway if it's expected, I can move this to Zephyr tree.

Reply:
see #3